apps/typegpu-docs/src/content/docs/fundamentals/roots.mdx

```diff
@@ -59,7 +59,8 @@ untyped value of a typed resource, use the `root.unwrap` function.
 
 ## Destroying resources
 
-Calling `root.destroy()` will destroy all resources created with it.
+Calling `root.destroy()` will destroy all resources created with it. 
+It will also destroy the underlying WebGPU device, if it wasn't originally passed in via the `initFromDevice` function.
```

Comment:
Only true once we release 0.3
We could either postpone the merging of this PR until then, or only add this line of documentation to the 0.3 branch.
```suggestion
```

Reply:
i choose number 2